### DISCUSSION TITLE
Can't open MAS after a lag issue

### DISCUSSION THREAD

**Discussion by TheSansation** [Monika-After-Story/MonikaModDev#8328] [QA - tech support + general] [answered]
After lagging with the game open, i reseted it (I was doing a update) and when it reopened, i had the Graphic performance error screen, i tried to ignore it, but the game stopped responding and i had to reset it again, when i tried to open it again tho, it opened a notepad file with a error message (close to the ones you can ignore in game) but would not open the actual game, this is the second time it happens and i don't want to reinstall the game yet again.      
The file:
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/7532725/traceback.txt)

**Comment by ibm2431** [comment, marked as answer]
#8193

`Monika After Story 0.12.3`

> [Update your game manually. MAS is currently 0.12.4. You might've downgraded which caused some issues or you have some mismatching files from different versions.](https://github.com/Monika-After-Story/MonikaModDev/issues/8193#issuecomment-962546900)

**Comment by multimokia** [comment]
Couple issues.
1. As pointed out above, you're running 0.12.3 and it looks like you've run 0.12.4 before
2. You're running from a zip. This is likely why the version rolled back, zip files don't run where you think they do. Extract the zip to somewhere like your desktop
